src/_includes/partials/footer.html

```diff
@@ -8,6 +8,7 @@
 				Contact Us:
 				<a class="email" href="mailto:gig@bytezone.dev">
 					gig@bytezone.dev
+					<iconify-icon icon="bi:send"></iconify-icon>
```

Comment:
[FA has envelope icons](https://icones.js.org/collection/fa6-regular?s=envelope) that would work here.

For example,

```suggestion
					<iconify-icon icon="bi:send"></iconify-icon>
					<iconify-icon icon="fa6-regular:envelope"></iconify-icon>
```